notifications/rest/notifications/send-notification-detailed/send-notification-detailed.py

```diff
@@ -0,0 +1,13 @@
+# Download the Python helper library from twilio.com/docs/python/install
+from twilio.rest import Client
+
+# Your Account Sid and Auth Token from twilio.com/user/account
+account = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+token = "{{ auth_token }}"
+client = Client(account, token)
+
+service = client.notifications.v1.services("ISxxx")
+
+notification = service.notifications.create(identity="Bob", title="Generic loooooooong title for all Bindings", body="This is the body for all Bindings", gcm='{"notification":{"tag":"MyTag"}}', apn='{"aps":{"title":"Short title for Watch."}}')
```

Comment:
@jefflinwood line is too long. Please refer to the [PEP8](https://www.python.org/dev/peps/pep-0008/#indentation) style guide to split this in multiple lines.